pkgs/development/libraries/libsoup/3.x.nix

```diff
@@ -22,25 +25,23 @@ stdenv.mkDerivation rec {
   pname = "libsoup";
   version = "3.2.2";
 
-  outputs = [ "out" "dev" "devdoc" ];
+  outputs = [ "out" "dev" ] ++ lib.optional withIntrospection "devdoc";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
     sha256 = "sha256-g2c8aFuRD7fTnx8o7uWvvvtxwFeY/DUKw78biF4e+qE=";
   };
 
-  depsBuildBuild = [
-    pkg-config
-  ];
```

Comment:
`pkgsCross.aarch64-multiplatform.libsoup_3`

```
libsoup-aarch64-unknown-linux-gnu> Build-time dependency gi-docgen found: NO (tried pkgconfig and cmake)
libsoup-aarch64-unknown-linux-gnu> Looking for a fallback subproject for the dependency gi-docgen
libsoup-aarch64-unknown-linux-gnu> docs/reference/meson.build:14:0: ERROR: Automatic wrap-based subproject downloading is disabled
```

Reply:
Whoops, thanks for catching.